OCPBUGS-16184: Fix QEMU link (#62430)
openshift/openshift-docs: modules/virt-adding-virtio-drivers-vm-yaml.adoc

[id="virt-adding-virtio-drivers-vm-yaml_{context}"]
= Adding VirtIO drivers container disk to a virtual machine

> Add a VirtIO drivers container disk to a Windows virtual machine (VM) as a SATA CD drive.

Reviewer: This is a little bare. From the mod docs guidance:

> The procedure introduction is a short paragraph that provides an overview of the module. The introduction includes what the module will help the user do and why it will be beneficial to the user. It includes key words that relate to the module to maximize search engine optimization.
>
> The introduction typically provides context for the procedure, such as:
> 
>   * Why and where the user performs the procedure
>   * Special considerations specific to the procedure

.Prerequisites

* Download the `container-native-virtualization/virtio-win` container disk from the
link:https://catalog.redhat.com/software/containers/search?q=virtio-win&p=1[Red Hat Ecosystem Catalog].
+
This is not mandatory, because the container disk is downloaded from the Red Hat registry

> if it not already present in the cluster. However, downloading reduces the installation time.

Reviewer: Could this be a tip instead? It's not exactly a prereq if the procedure will work fine without it.

      image: container-native-virtualization/virtio-win
    name: virtiocontainerdisk
----

> <1> {VirtProductName} boots the VM disks in the order defined in the `VirtualMachine` manifest. You can define other VM disks that boot before the `container-native-virtualization/virtio-win` container disk or use the optional `bootOrder` parameter to ensure the VM boots from the correct disk. If you configure the boot order for a disk, you must configure the boot order for the other disks.

Reviewer 2: ```suggestion
<1> {VirtProductName} boots the VM disks in the order defined in the `VirtualMachine` manifest. You can either define other VM disks that boot before the `container-native-virtualization/virtio-win` container disk, or use the optional `bootOrder` parameter, to ensure the VM boots from the correct disk. If you configure the boot order for a disk, you must configure the boot order for the other disks.
```

Reviewer 2: Out of scope for this PR, but you might consider clarifying this footnote.

Author: I will add  "either/or" but the commas are unnecessary, IMO.

. Apply the changes:

> * If the VM is running, run `oc apply -f <vm.yaml>` or reboot the VM.

Reviewer 2: ```suggestion
* If the VM is running, reboot the VM or run the following command:
+
[source, terminal]
----
$ run `oc apply -f <vm.yaml>`
----
```

Reviewer 2: Not sure if you have a different style in the Virualization docs, but current OCP style is to list commands like this so they are easy for the customer to copy. For your consideration.

Author: Module was written 4 years ago, long before current OCP style. This is outside the scope of the change but I can try to clean it up.

> * If the VM is not running, run `virtctl start <vm>`.

Reviewer 2: ```suggestion
* If the VM is not running, run the following command:
+
[source, terminal]
----
 $ `virtctl start <vm>`
----
```

Reviewer: I'd probably go with "If the VM is not running, start it by running the following command:" or something like that.

> . After the virtual machine has started, install the VirtIO drivers from the SATA CD drive.

Reviewer 2: ```suggestion
. After the virtual machine starts, you can install the VirtIO drivers from the SATA CD drive.
```
